source/common/http/conn_manager_impl.h

```diff
@@ -245,15 +288,21 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
       }
       getSavedRequestMetadata()->clear();
     }
+
     // This function is called after the filter calls decodeHeaders() to drain accumulated metadata.
     void handleMetadataAfterHeadersCallback() override;
 
     // Http::StreamDecoderFilterCallbacks
     void addDecodedData(Buffer::Instance& data, bool streaming) override;
+
```

Comment:
could you use clang-format and remove all the blanks added?

Reply:
Hmm. This *is* clang-format fixing formatting.